doc/tutorial.txt

```diff
@@ -489,6 +489,10 @@ taskB ...-------------------|resume taskA|run 10    |--------------------
 * yield: String. Calls pthread_yield(), freeing the CPU for other tasks. This has a
 special meaning for SCHED_DEADLINE tasks. String can be empty.
 
+* fork: String. Instead of creating all tasks at start time, you can fork one
```

Comment:
I thought i made a comment on "instance":0 but can't find it anymore. So this comment maybe duplicated with another one:
You should update the description of "instance" for saying that "instance": 0 will now create the data of the task but will not create the thread. This is a change compare to current behavior. at now "instance":0 can be used to remove a task for the use case

Reply:
yeah same here I received the emails but took me a while before I could see the actual comments on the page - github glitch probably..